class1/README.md

```diff
@@ -1 +1,131 @@
 #Class 1
+
+##Goals: 
+- Students should understand course plan and resources
+- Development environments ready
+- Fundamentals of Node and JavaScript
+
+##Homework Due:
+For this first class you should:
+- Have a working instance of linux (probably Ubuntu). Mac users will probably be able to get by with OS X. Note that we will officially be doing everything on Ubuntu.
+- Have Node installed. Do this on Ubuntu using ```sudo apt-get install -y nodejs ``` and on Mac using homebrew  ```brew install node ```.
```

Comment:
Rather than give specifics for how to do this, link to the `class0` folder which will give the guide for this homework.
Also, you can't just apt-get install node without adding a PPA, the base apt-get repositories are locked to a pretty old version of node.